src/main/resources/mustache-templates/phenotypes.html

```diff
@@ -0,0 +1,99 @@
+{{>layout/header}}
+
+<div class="container vertical-space">
+	<h3>Patient Info</h3>
+	<ul class="list-unstyled">
+		{{#patient}}
+		<li>Name: <span>{{firstName}} {{lastName}}</span></li>
+		<li>Id: <span id="patient_id">{{id}}</span></li> 
+		{{/patient}}
+	</ul>
+	<h3>Phenotypes</h3>
+	<table id="hposummary" class="table table-bordered" style="width: 100%">
+		<thead class="thead-light">
+			<tr>
+				<th></th>
+				<th>HPO Term Name</th>
+				<th>HPO Term Id</th>
+				<th># Observations</th>
+				<th>First Date</th>
+				<th>Last Date</th>
+			</tr>
+		</thead>
+	</table>
+</div>
+
+
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/jquery/jquery.min.js"></script>
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/datatables/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript"
+	src="{{req.contextPath}}/webjars/datatables/js/dataTables.bootstrap4.min.js"></script>
```

Comment:
Are these included in the footer as well?

Reply:
No, this is the only js at this point though I may want to add jquery to highlight navigation elements. I haven't found a great approach for this. I know we want to load js last if possible, but we may want to run scripts on pages that need it. How are you handling in CDCA?